plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -365,6 +390,11 @@ public static Map<String, String> getColumnInvariants(MetadataEntry metadataEntr
         return getColumnProperties(metadataEntry, DeltaLakeSchemaSupport::getInvariants);
     }
 
+    public static Map<String, Map<String, Object>> getColumnsMetadata(MetadataEntry metadataEntry)
```

Comment:
nit: Could you move to after `getInvariants()`? The current place is between `getColumnInvariants()` and `getInvariants()`.